src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -541,6 +541,14 @@ void WriteCryptoFrame(
         0, ClientContext->State.BufferLength, ClientContext->State.Buffer
     };
 
+    //
+    // TODO: The code in the recvfuzzer assumes that all data produced in
```

Comment:
Was this TODO in the original PR? It seems unrelated to the fix.

Reply:
No, but I wanted to keep this comment rather than leave it out when resolving the merge conflict.

More docs = better anyway.

Reply:
That's a bit more complicated. More good documentation is better.
But if you pick a TODO / comment outside of the context it was written in, it might not be true or relevant for this branch.

IMO, we should pick this TODO if and when we backport the PR that introduced it.